cmd/frontend/graphqlbackend/survey_response.go

```diff
@@ -56,6 +56,18 @@ func (s *surveyResponseResolver) Better() *string {
 	return s.surveyResponse.Better
 }
 
+func (s *surveyResponseResolver) UseCases() *[]string {
+	return &s.surveyResponse.UseCases
+}
```

Comment:
It is nullable in the API but here it'll always return at least an empty `[]`. Is this intentional? If you want to return `nil` you have to do 

```suggestion
func (s *surveyResponseResolver) UseCases() *[]string {
if len(s.surveyResponse.UseCases) == 0 {
return nil
}
	return &s.surveyResponse.UseCases
}
```

Otherwise we might consider just making it non-nullable in the schema as well?